synapseAnnotations/data/sageCommunity.json

```diff
@@ -519,6 +519,21 @@
         "value": "mtx",
         "description": "Matrix Market Exchange Format",
         "source": "https://math.nist.gov/MatrixMarket/formats.html#MMformat"
+      },
+      {
+        "value":"bfilt",
+        "description": "contains the final ATACSeq peaks after removing the encode blacklist regions.",
+        "source":"https://sites.google.com/site/anshulkundaje/projects/blacklists"
```

Comment:
Is this really a file format, or are [these files](https://github.com/Boyle-Lab/Blacklist/tree/master/lists) (are they going to be hosted on Synapse?) just BED files with specific contents?

Reply:
The file path is actually .bfilt (for example, https://www.synapse.org/#!Synapse:syn18359515)

Reply:
The file format and file extension are not necessarily the same thing, so I guess I'm trying to understand what about these files warrants its own format. Is there something about the structure or format of these files that is unique and standardized?

Reply:
In the case of the bfilt, the file structure does not appear to deviate from a bed file so I will remove value.